web/Views/Admin/TenantMatchList.cshtml

```diff
@@ -0,0 +1,65 @@
+﻿@using BricksAndHearts.Services
+@model TenantMatchListModel
+
+@{
+    ViewData["Title"] = "Tenant List";
+}
+
+@{
+    if (TempData["MultipleFlashMessages"] != null)
+    {
+        var flashMessages = TempData["MultipleFlashMessages"] as string[];
+        var flashTypes = TempData["MultipleFlashTypes"] as string[];
+        for (int i = 0; i < flashMessages!.Length; i++)
+        {
+            <div class="alert alert-@flashTypes![i]" role="alert">
+            @flashMessages[i]
+            </div>
+        }
+    }
+}
```

Comment:
if (TempData["MultipleFlashMessages"] != null) section here shouldn't be necessary any more.